### DISCUSSION TITLE
Zoom in and out for non-PDF files

### DISCUSSION THREAD

**Discussion by gutzsant** [sumatrapdfreader/sumatrapdf#2505] [Ideas]
I'd like to see a zoom function for non-PDF files. For example, while font size can be adjusted in EPUB files to make the font larger, the same can't be done for any embebed image. This a a big flaw when reading with Sumatra when there are maps, character profiles or illustrations in novels, particularly novelettes or light novels. If only the zoom worked as it does with PDF files, with the + and - keys, that would be enough.

**Comment by GitHubRulesOK** [comment]
use the fixed interface `UseFixedPageUI = true` that has zoom and search in version 3.2 that  was easier from menu in 3.3 or 3.1 its the old manual method
```
EbookUI [
	FontName = Georgia
	FontSize = 12.5
	TextColor = #5f4b32
	BackgroundColor = #fbf0d9
	UseFixedPageUI = true
]
```
in 3.4 pre-release its no longer needed for most formats but there may be exceptions ?

**Reply by gutzsant** [reply]
Thanks, that works. However, I'd still like to see that feature fully implemented for non-fixed interface because that forces a black font with white background view and I prefer a dark theme, which is not possible when using `UseFixedPageUI = true` as far as I know. Is a dark theme with zoom possible in ver. 3.4?

**Reply by GitHubRulesOK** [reply]
There are currently NO theme features you will see that the [canvas behavior](https://github.com/sumatrapdfreader/sumatrapdf/issues/853) should not be altered except for [intentional reversal](https://github.com/sumatrapdfreader/sumatrapdf/issues/1275) via Shift+I (shift eye)
and scrollbars topbar is controled by windows thus  in the past worked well with XP themes https://github.com/sumatrapdfreader/sumatrapdf/issues/898
but MS moves theming in different directions per update, 
The tabs and scrollbars are modified by [theming applications](https://github.com/sumatrapdfreader/sumatrapdf/discussions/2441)

**Reply by gutzsant** [reply]
It isn't bothering that the windows and scrollbars are in regular colors. What's bothering is being unable to change the background and font colors of an ebook without disabling zooming in on embedded images. Alternatively, I hope it can eventually be possible to make the font and background of epub files a darker tone without the color of the embedded images getting altered when modifying `TextColor =` and `BackgroundColor =` in `FixedPageUI [`.
